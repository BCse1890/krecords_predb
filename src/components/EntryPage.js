import React from 'react';
import RecordList from './RecordList';
import RecordListFilters from './RecordListFilters';

const EntryPage = () => (
    <div>
        <RecordListFilters />
        <RecordList/>
    </div>
);


export default EntryPage;