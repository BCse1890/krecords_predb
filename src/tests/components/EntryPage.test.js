import React from 'react';
import { shallow } from 'enzyme';
import '../setUpTests';
import EntryPage from '../../components/EntryPage';

test('should render Entrypage', () => {
    const wrapper = shallow(<EntryPage />);
    expect(wrapper).toMatchSnapshot();
})