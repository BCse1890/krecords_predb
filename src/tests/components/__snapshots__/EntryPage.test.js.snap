// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Entrypage 1`] = `
<div>
  <Connect(RecordListFilters) />
  <Connect(RecordList) />
</div>
`;
